ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,126 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "EmailAddress",
+    "ListElementCode": "email",
+    "FieldType": "Text",
+    "ElementLabel": "Email",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "contactDirection",
+    "FieldType": "Text",
+    "ElementLabel": "Person to contact",
```

Comment:
have you checked these fields through with content. I am currently going through potentially needing to add all these telephone fields to respondent, so would be interested to know what they think

Reply:
<img width="1019" alt="Screenshot 2019-07-10 at 15 13 38" src="https://user-images.githubusercontent.com/42037338/60976296-55327b80-a325-11e9-9e84-1aa336cc0243.png">
So i was just going by this? What do you think? @swalker125